defaults/main.yml

```diff
@@ -13,6 +13,8 @@ pretalx_database_port: 5432
 pretalx_staticfiles_directory: /usr/share/webapps/pretalx{{ pretalx_instance_identifier }}/static/
 pretalx_data_dir: /usr/share/webapps/pretalx{{ pretalx_instance_identifier }}/data/
 pretalx_media_dir: /usr/share/webapps/pretalx{{ pretalx_instance_identifier }}/media/
+pretalx_python: "{{ pretalx_virtualenv }}/bin/virtualenv"
```

Comment:
Would prefer these two to be the other way around, just for ease of reading/logical order.